### PR TITLE
Feature/implement_evaluation_controller_be

### PR DESCRIPTION
**Pull request by VladyslavCherednychenko**
make ef migration to add some new data just for test 
add evaluation controller

### CODE REVIEW

File: Portal_Perfomance_Employes/Controllers/UserEvaluationController.cs [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+        [Route("GetUserHardSkills")]
+        [HttpGet]
+        public async Task<IActionResult> GetUserHardSkills()
+        {
+            int.TryParse(User.FindFirst(ClaimTypes.NameIdentifier)?.Value, out int id);
+            var empSkills = await _context.EmployeeSkills.Where(e => e.EmployeeId == id).ToListAsync();
+            if (empSkills != null)
+            {
+                var hardSkillsTable = new DataTable();
```
Comment (DmytroDorodnykhTeamInternational) [inline Portal_Perfomance_Employes/Controllers/UserEvaluationController.cs]: can be moved to a func, same code

File: Portal_Perfomance_Employes/Controllers/UserEvaluationController.cs [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+                return Ok(hardSkillsTable.ToJson());
+            }
+            return NotFound("The user skills list is empty");
+        }
+
+        [Route("AddUserSkill")]
+        [HttpPut]
+        public async Task<IActionResult> AddUserSkill(int skillType, string skillName, int skillLevelId, int skillLevelTypeId)
+        {
+            int.TryParse(User.FindFirst(ClaimTypes.NameIdentifier)?.Value, out int id);
```
Comment (DmytroDorodnykhTeamInternational) [inline Portal_Perfomance_Employes/Controllers/UserEvaluationController.cs]: if fails to parse and 0 by default? could cause some issues if we have something with id 0 in db

**Review by DmytroDorodnykhTeamInternational — APPROVED** [review]

File: Portal_Perfomance_Employes/Data/AppDbContext.cs [hunk 3]
```diff
@@ -230,6 +230,26 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                 new TeamMember { Id = 10, EmployeeId = 11, TeamId = 2, FirstName = "Emp11", LastName = "Test11", FromDate = DateTime.Now, IsActive = true },
                 new TeamMember { Id = 11, EmployeeId = 4, TeamId = 2, FirstName = "Emp4", LastName = "Test4", FromDate = DateTime.Now, IsActive = true }
                 );
+            modelBuilder.Entity<EmployeeSkill>().HasData(
```
Comment (dangoth) [inline Portal_Perfomance_Employes/Data/AppDbContext.cs:233]: Can you use data seeding from my last PR so we don't have double population of the same IDs or conflicts when merging? Alternatively I can overwrite mine

**Review by dangoth — APPROVED** [review]

File: Portal_Perfomance_Employes/Controllers/UserEvaluationController.cs (hunk 2 above)
Comment (VladyslavCherednychenko) [reply Portal_Perfomance_Employes/Controllers/UserEvaluationController.cs]: will be done! Also nooticed that I made same mistake in other controller so I'll fix them also

File: Portal_Perfomance_Employes/Controllers/UserEvaluationController.cs (hunk 1 above)
Comment (VladyslavCherednychenko) [reply Portal_Perfomance_Employes/Controllers/UserEvaluationController.cs]: will be done

File: Portal_Perfomance_Employes/Data/AppDbContext.cs (hunk 3 above)
Comment (VladyslavCherednychenko) [reply Portal_Perfomance_Employes/Data/AppDbContext.cs:233]: Thanks for the notice, I'll do it in a new commit

**Review by Ygoraphael — APPROVED** [review]

**Pull request merged**